### PR TITLE
Hide channel from version for tagged commits

### PR DESCRIPTION
**Pull request by Scrumplex**
With this PR, our official builds from GHA should not include the `-<commit sha>` suffix.

### CODE REVIEW

File: buildconfig/BuildConfig.cpp.in [hunk 1]
```diff
@@ -62,8 +62,10 @@ Config::Config()
     UPDATER_BASE = "@Launcher_UPDATER_BASE@";
 
     GIT_COMMIT = "@Launcher_GIT_COMMIT@";
+    GIT_TAG = "@Launcher_GIT_TAG@";
     GIT_REFSPEC = "@Launcher_GIT_REFSPEC@";
-    if (GIT_REFSPEC == QStringLiteral("GITDIR-NOTFOUND"))
+    if (GIT_REFSPEC == QStringLiteral("GITDIR-NOTFOUND")
+        || GIT_TAG == QStringLiteral("-128-NOTFOUND"))
     {
         VERSION_CHANNEL = QStringLiteral("stable");
```
Comment (kthchew) [inline buildconfig/BuildConfig.cpp.in]: It's not really accurate for a build from a pull request to be labeled as part of the "stable" channel, though, is it? Although, is there a way to easily tell that a build was made from a PR as opposed to a release?

Screen Shot 2022-06-22 at 6 55 06 PM

Either way, the channel looks like it's pretty much unused in PolyMC anyway, so 🤷🏼‍♂️

Comment (Scrumplex) [reply buildconfig/BuildConfig.cpp.in]: oh hm.. I thought it only is -128-NOTFOUND, when there is no Git repo

Comment (kthchew) [reply buildconfig/BuildConfig.cpp.in]: but now the tag's not hidden :/
Screen Shot 2022-06-23 at 11 58 49 PM

Comment (Scrumplex) [reply buildconfig/BuildConfig.cpp.in]: Works fine for me. Did you tag the current commit locally?

Comment (kthchew) [reply buildconfig/BuildConfig.cpp.in]: > Did you tag the current commit locally?

Yeah oops that was the problem, ignore me lol

**Comment by DioEgizio** [comment]
image


doesn't seem to work :P

**Comment by DioEgizio** [comment]
oh nvm this is for tagged commits built from gha, not also devbuilds built from gha, ignore me

**Review by ZekeZDev — APPROVED** [review]

**Review by flowln — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Pull request merged**